conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```diff
@@ -11,7 +11,15 @@ jobs:
     fetchDepth: {{ clone_depth }}
 {%- endif %}
   - script: |
-         rm -rf /opt/ghc
+         sudo rm -rf /opt/ghc \
+                     /opt/hostedtoolcache/CodeQL \
```

Comment:
What else is in the `/opt/hostedtoolcache` directory?

Reply:
Yeah wondering if we should delete this whole directory. Those seem to be conveniences for users wanting to just run CI with one of the preinstalled languages, which we don't need

Reply:
The timing results deletes everything in `hostedtoolcache`.

Reply:
```suggestion
```